NEWS.md

```diff
@@ -27,6 +28,141 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## [unreleased]
+
+This release is shaping up to be the second in a row that is larger than
+usual both in terms of new features and fixes. As usual, most of the
```

Comment:
Should we point out the input method works (incl popups) here specifically?

Reply:
Have added `input methods`. Good point.